JavaIntegration/src/main/java/org/openzen/zencode/java/module/JavaNativeTypeTemplate.java

```diff
@@ -194,18 +197,42 @@ private void loadMethods() {
 				continue;
 
 			FunctionHeader header = headerConverter.getHeader(typeVariableContext, method);
-
-			if (expansion && !isStaticExpansion) {
+			if (expansion && !isStaticExpansionMethod) {
 				FunctionParameter[] withoutFirst = Arrays.copyOfRange(header.parameters, 1, header.parameters.length);
 				header = new FunctionHeader(header.typeParameters, header.getReturnType(), header.thrownType, withoutFirst);
 			}
 
 			for (MethodID id : ids) {
-				JavaRuntimeMethod runtimeMethod = new JavaRuntimeMethod(class_, target, method, id, header, implicit, expansion && !isStaticExpansion);
+				JavaRuntimeMethod runtimeMethod = new JavaRuntimeMethod(class_, target, method, id, header, implicit, expansion && !isStaticExpansionMethod);
 				methods.computeIfAbsent(id, x -> new ArrayList<>()).add(runtimeMethod);
 				class_.module.getCompiled().setMethodInfo(runtimeMethod, runtimeMethod);
 			}
 		}
+
+
+		if (class_.cls.isEnum()) {
+			Stream.of(
+					BuiltinMethodSymbol.ENUM_NAME,
+					BuiltinMethodSymbol.ENUM_ORDINAL,
+					//BuiltinMethodSymbol.ENUM_VALUES,
```

Comment:
This symbol was moved down to the try-catch and thus registers the method natively instead of using the BuiltinMethodSymbol. Reason was that the function header in the method symbol is the `FunctionHeader.PLACE_HOLDER` which is never resolved to the actual header.

Should the other 3 symbols (name, ordinal, compare) also be registered natively, or should they use the BuiltinMethodSymbols?